### DISCUSSION TITLE
Binary-only previewer

### DISCUSSION THREAD

**Discussion by J-Kappes** [gokcehan/lf#1641] [Ideas]
I noticed a difference in latency previewing text files between the default and mailcap-print-less as per the Wiki tutorial. Would it make sense to provide an option to check if it is a text file first and only run the custom previewer if it isn't?

I imagine that would considerably cut the latency difference when previewing text files.

**Comment by zongou** [comment]
https://github.com/gokcehan/lf/discussions/1686
